### DISCUSSION TITLE
create-tiddler-without-write-access-and-try-to-comment

### DISCUSSION THREAD

**Discussion by giscus** [abesamma/giscus-for-tiddlywiki-demo#5] [Announcements]
# create-tiddler-without-write-access-and-try-to-comment

https://abesamma.github.io/giscus-for-tiddlywiki-demo/

**Comment by pmario** [comment]
Hi I did create tiddler named: create-tiddler-without-write-access-and-try-to-comment and the system opens a comment input. .. So let's test if it is possible to use it that way.
